server/src/weaverbird/backends/pypika_translator/translators/googlebigquery.py

```diff
@@ -59,14 +59,10 @@ def _get_single_condition_criterion(
 
         if isinstance(condition, DateBoundCondition):
             match condition.operator:
-                case "from":
-                    return functions.Cast(column_field, "datetime") >= ParseDatetime(
-                        "%FT%T", condition.value
-                    )
                 case "until":
-                    return functions.Cast(column_field, "datetime") <= ParseDatetime(
-                        "%FT%T", condition.value
-                    )
+                    return functions.Cast(column_field, "datetime") >= condition.value
+                case "from":
+                    return functions.Cast(column_field, "datetime") <= condition.value
```

Comment:
This seems to do the opposite of what we want, I don't understand the reason behind this. Could you please provide an example of a query where `UNTIL {date}` returns everything **after** the date ?

Reply:
👍🏾 https://github.com/ToucanToco/weaverbird/pull/1413#discussion_r919021452